styx-scheduler-service/src/main/java/com/spotify/styx/StyxScheduler.java

```diff
@@ -460,6 +457,10 @@ public void create(Environment environment) {
     final Duration stateManagerTickInterval = get(config, config::getDuration, STYX_STATE_MANAGER_TICK_INTERVAL)
         .orElse(DEFAULT_STATE_MANAGER_TICK_INTERVAL);
 
+    // TODO: is the shutdown timeout of 1 second here sane?
```

Comment:
Can you add a comment so that future us know that closing order is important?. Something like:

    // Closer is a FIFO queue for closing resources and we need to make sure that we close the 
    // ScheduledExecutorService last so ....

Reply:
It is a stack so its LIFO